src/modules/Dashboard/features/Applet/Activities/ActivitiesToolbar/ActivitiesToolbar.tsx

```diff
@@ -19,6 +21,10 @@ export const ActivitiesToolbar = ({
   const { t } = useTranslation('app');
   const { featureFlags } = useFeatureFlags();
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const workspaceRoles = workspaces.useRolesData();
+  const roles = appletId ? workspaceRoles?.data?.[appletId] : undefined;
+
```

Comment:
Extremely nit-picky, and I'm not opposed to linebreaks for readability or whatever, but is the extra line break here intentional? 🧐